airframe-http-recorder/src/test/scala/wvlet/airframe/http/recorder/HttpRecorderTest.scala

```diff
@@ -71,7 +71,7 @@ class HttpRecorderTest extends AirSpec {
     response.status shouldBe replayResponse.status
     replayResponse.headerMap.get("X-Airframe-Record-Time") shouldBe defined
     orderInsensitveHash(response.headerMap.toMap) shouldBe orderInsensitveHash(
-      (replayResponse.headerMap - "X-Airframe-Record-Time").toMap
+      (replayResponse.headerMap -= "X-Airframe-Record-Time").toMap
```

Comment:
`- key ` ... `clone() -= key` ... is deprecated, but `-=` is OK.
`-=` is a mutator method which is not compatible with `-`, but for this test it seems OK.